pkgs/development/libraries/hfst/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, autoconf
+, automake
+, autoreconfHook
+, bison
+, flex
+, fetchFromGitHub
+, gettext
+, icu
+, libtool
+, stdenv
+, swig
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hfst";
+  version = "3.16.0";
+
+  src = fetchFromGitHub {
+    owner = "hfst";
+    repo = "hfst";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-2ST0s08Pcp+hTn7rUTgPE1QkH6PPWtiuFezXV3QW0kU=";
+  };
+  nativeBuildInputs = [
+    autoconf
+    automake
+    autoreconfHook
+    bison
+    flex
+    libtool
+    pkg-config
+    swig
```

```suggestion
    autoreconfHook
    bison
    flex
    pkg-config
    swig
```
those are included in autoreconfHook